.gitignore

```diff
@@ -27,7 +27,6 @@
 /config/master.key
 
 /test/*
-
+.rake_tasks~
```

Comment:
I do not understand why you need this?

Reply:
That file started popping up and I thought it was unnecessary.